lib/routes/linkedin/posts.ts

```diff
@@ -0,0 +1,59 @@
+import { Route } from '@/types';
+import puppeteer from '@/utils/puppeteer';
+import { parseCompanyName, parseCompanyPosts, BASE_URL } from './utils';
+import logger from '@/utils/logger';
+
+export const route: Route = {
+    path: '/company/:company_id/posts',
+    categories: ['social-media'],
+    example: '/linkedin/company/google/posts',
+    parameters: { company_id: "Company's LinkedIn profile ID" },
+    description: "Get company's LinkedIn posts by company ID",
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportRadar: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: 'Company Posts',
+    maintainers: ['saifazmi'],
+    handler: async (ctx) => {
+        const company_id = ctx.req.param('company_id');
+
+        // Puppeteer setup
+        const browser = await puppeteer();
+        const page = await browser.newPage();
+        await page.setRequestInterception(true);
+
+        page.on('request', (request) => {
+            request.resourceType() === 'document' ? request.continue() : request.abort();
+        });
+
+        const url = new URL(`${BASE_URL}/company/${company_id}`);
+
+        logger.http(`Requesting ${url.href}`);
+        await page.goto(url.href, {
+            waitUntil: 'domcontentloaded',
+        });
+
+        const response = await page.content();
+        page.close();
```

Comment:
```suggestion
        await page.close();
        await browser.close();
```

Reply:
Happy to make the change, but this code comes from https://docs.rsshub.app/joinus/new-rss/start-code#using-puppeteer

maybe this also needs to be updated there?

Reply:
Hi, thanks for the review @TonyRL , happy to make the change, but this bit of code comes from the documentation here: https://docs.rsshub.app/joinus/new-rss/start-code#using-puppeteer

should that also be updated?

Reply:
fixed in recent commits